packages/react-menu/Spec.md

```diff
@@ -929,106 +930,41 @@ The below references were used to decide n appropriate keyboard interactions fro
-
-As a general rule, once a submenu is dismissed without dismissing the menu, the focus should revert to the triggering menu item unless the interaction involves another focusable UI component.
-
-| Type     | Action     | Result  | Details                                                          | Focus after                                        |
-| -------- | ---------- | ------- | ---------------------------------------------------------------- | -------------------------------------------------- |
-| Mouse    | Click      | Open    | Click the menu item that contains a submenu                      | First menuitem in submenu                          |
-| Mouse    | Hover      | Open    | Hover over the menu item that contains a submenu with delay      | First menuitem in submenu                          |
-| Keyboard | Enter      | Open    | Focus on triggering menu item                                    | First menuitem in submenu                          |
-| Keyboard | Space      | Open    | Focus on triggering menu item                                    | Frist menuitem in submenu                          |
-| Keyboard | ArrowRight | Open    | Focus on triggering menu item                                    | First menuitem in submenu                          |
-| Mouse    | Click      | Dismiss | Click on an item in the submenu                                  |                                                    |
-| Keyboard | Space      | Dismiss | Invoked on a submenu item                                        |                                                    |
-| Keyboard | Space      | Dismiss | Invoked on a submenu item                                        |                                                    |
-| Mouse    | Click      | Dismiss | Click on a UI element that is not the submenu                    | Root menu trigger                                  |
-| Mouse    | MouseLeave | Dismiss | Mouse leaves the submenu or its triggering menu item after delay | Root menu trigger                                  |
-| Keyboard | ArrowLeft  | Dismiss | Closes the submenu                                               | menu item that contained submenu                   |
-| Keyboard | Esc        | Dismiss | Closes the submenu                                               | menu item that contained submenu                   |
-| Keyboard | Tab        | Dismiss | Closes the menu and all submenus                                 | Next tabbable element after root menu trigger      |
-| Keyboard | Shift+Tab  | Dismiss | Closes the menu and all submenus                                 | Previous tabbable element before root menu trigger |
+### Mouse/Keyboard interactions
+
+Below is a set of diagrams that tries to illustrates all the interactions menus and nested menus support in an easily understandable way.
+
+> TODO convert these diagrams to excalidraw or smth that is text format
```

Comment:
Just a text format of the image or also a textual explanation of what happened? (similar to what has been in the tables this PR is removing)

Reply:
> Just a text format of the image
That was the primary goal of the TODO
so I did these originally in PPT because that's what I was familiar with. But now that I'm checking in stuff to github text format is better for maintenance


> or also a textual explanation of what happened

Yeah that's a good point, I will modify the TODO to add more descriptions, will be a lot easier once it's possible to use a verionable drawing tool.